recipes/pyne/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = "0.5.1" %}
+{% set org = "pyne" %}
+{% set branch = "0.5.1" %}
+{% set sha256 = "4028bbe10e6f74d38af2493e2625a77dda2e275a1bf85e2a9b08df56260cdcd8" %}
+{% set variant = "openblas" %}
+
+package:
+  name: pyne
+  version: {{ version }}
+
+source:
+  fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/{{ org }}/pyne/archive/{{ branch }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or (osx and py34)]
+  detect_binary_files_with_prefix: true
+  features:
+    - blas_{{ variant }}
+
+#
+# note that builds timeout on mac with many versions of python & numpy
+#
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - pkg-config
+    - setuptools
+    - gcc
+    - libgcc
+    - libgfortran
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - moab
+    - cython
+    - numpy x.x  # [linux]
+    - numpy 1.11.*  # [osx]
+    - pytables
+  run:
+    - python
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.18*
+    - boost 1.61.*
+    - hdf5 1.8.17|1.8.17.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - moab
+    - cython
+    - numpy x.x  # [linux]
```

Comment:
See comment above.  The Travis resources are too slow to compile more than 2 versions on Mac without timing out.